python3/libexec/nbd_client_manager.py

```diff
@@ -206,7 +212,17 @@ def connect_nbd(path, exportname):
                     "-name",
                     exportname,
                 ]
-                _call(cmd)
+                ret = _call(cmd, raise_err=False, log_err=True)
+                if NLE_BUSY == ret:
```

Comment:
FYI. The related PR on nbd-client is https://github.com/NetworkBlockDevice/nbd/pull/171

Reply:
yes, it is good enough to resolve the issue described here.
However, technically, nbd-client can still conflict with other process other than systemd-udevd, 
so it still good to have this patch.